lib/matplotlib/stackplot.py

```diff
@@ -26,7 +26,7 @@ def stackplot(axes, x, *args, **kwargs):
     ----------
     x : 1d array of dimension N
 
-    y : 2d array of dimension MxN, OR sequence 1d arrays each of dimension 1xN
+    y : 2d array of dimension MxN, OR sequence of 1d arrays each of dimension 1xN
```

Comment:
This fails PEP8 because the line is too long and has caused downstream jobs to fail.  Not sure how to fix - easier to revert or just a new PR?

Reply:
I'll put in a new PR

Reply:
Thanks @dstansby